doc/api.rst

```diff
@@ -136,6 +136,7 @@ template and implement ``parse`` and ``format`` methods::
 
     // Parses format specifications of the form ['f' | 'e'].
     constexpr auto parse(format_parse_context& ctx) {
+    //auto parse(format_parse_context &ctx) -> decltype(ctx.begin()) //c++11
```

Comment:
Could you add a space after each `//` for consistency with other comments?

Thanks!

Reply:
added